src/main/java/xyz/kmahyyg/eshopdemo/controller/GeneralController.java

```diff
@@ -0,0 +1,12 @@
+package xyz.kmahyyg.eshopdemo.controller;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Controller
+public class GeneralController {
+
+    @RequestMapping("/")
+    public  String index() {return "index"; }
+
+}
```

Comment:
Useless file. Spring Boot already included a static resource handler, which will automatically handle this. Remove this file.